src/assets/styles/css/App.css

```diff
@@ -0,0 +1,32 @@
+.App {
```

Comment:
i think the team agreed in using `sass` and not `css`, please consider changing this

Reply:
On it.